core-bundle/src/Resources/contao/languages/en/tl_image_size.xlf

```diff
@@ -56,6 +56,21 @@
       <trans-unit id="tl_image_size.skipIfDimensionsMatch.1">
         <source>Skip image processing if resizing is not needed. Note that no optimizations such as meta data stripping or JPEG compression will be applied in this case.</source>
       </trans-unit>
+      <trans-unit id="tl_image_size.formats.0">
+        <source>Convert image formats</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.formats.1">
+        <source>Select which format conversion should be applied.</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.formatsMissingWebp">
+        <source>WEBP images are disabled. You can enable them by adding "webp" to contao.image.valid_extensions.</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.formatsOptions.{&quot;webp&quot;:[&quot;webp&quot;,&quot;jpg&quot;]}">
+        <source>WEBP to JPEG</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.formatsOptions.{&quot;jpg&quot;:[&quot;webp&quot;,&quot;jpg&quot;],&quot;jpeg&quot;:[&quot;webp&quot;,&quot;jpeg&quot;]}">
```

Comment:
Are you sure that our xliff reader understands keys with special characters?

Reply:
I tested it and the backend showed the correct labels.

Reply:
I'm just asking because we had to add a separate routine for keys with dots:

https://github.com/contao/contao/blob/99427681edf795b77ed29a172188e2c322d0629f/core-bundle/src/Config/Loader/XliffFileLoader.php#L130.

Reply:
I think dots are a special case because we are using them as an array path seperator.

Reply:
Yes.

I would still prefer a simple key in order to avoid possible issues in the future, but as long as it works, I am fine with the complex ones as well.

Reply:
We could also use a different format, any suggestions?

Somthing like `jpg:webp,jpg`?

Reply:
Yes, as simple as possible IMHO. 👍

Reply:
Done in 61accc2833ab1cdacbb0993763a7d632eb283a27